shared/js/smart-screen/scrollable.js

```diff
@@ -537,6 +543,30 @@
         'translateX(calc((100% + ' + this.spacing + 'rem) * ' + tabstop + '))';
     },
 
+    /**
+     * Shift node positions without animations.
+     * @param  {[type]} skipIdx The index of the node that needs not be shifted.
+     * @param  {[type]} offset  The unit offset. Use positive integer to shift
+     *                          right and negative integer to shift left.
+     */
+    _shiftOtherNodesPosition: function(skipIdx, offset) {
+      this.nodes.forEach(function(node, idx) {
+        if (idx !== skipIdx) {
+          node.style.transitionProperty = 'none';
+          var tabstop = this._getTabstop(idx) + offset;
+          this.getNodeFromItem(node).style.transform =
+            'translateX(calc((100% + ' +
+            this.spacing + 'rem) * ' + tabstop + '))';
+        }
+      }.bind(this));
+
+      getComputedStyle(this.nodes[0]).width;
```

Comment:
Let's wrap this `getComputedStyle` as a function like `forceReflow(elem)`.
And you can replace the `getComputedStyle` at the other place by `forceReflow`.